backend/app/api/v1/endpoints/conversations.py

```
@@ -0,0 +1,38 @@
+    conversations = get_conversations(db=db, skip=skip, limit=limit)
+    return conversations
+
+@router.get("/{conversation_id}", response_model=Conversation)
+def read_conversation(conversation_id: UUID, db: Session = Depends(get_db)):
+    db_conversation = get_conversation(db=db, conversation_id=conversation_id)
+    if db_conversation is None:
+        raise HTTPException(status_code=404, detail="Conversation not found")
+    return db_conversation
+
+@router.put("/{conversation_id}", response_model=Conversation)
+def update_existing_conversation(conversation_id: UUID, title: str, db: Session = Depends(get_db)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)